From b5ce993548d957f1a393d8c6b0ae5cd97e153542 Mon Sep 17 00:00:00 2001
From: Alex Zhao <zzc@rock-chips.com>
Date: Wed, 30 Dec 2020 11:07:21 +0800
Subject: [PATCH] arm64: configs: rockchip_defconfig: enable
 CONFIG_SDIO_KEEPALIVE

This is needed by RK356X EVB to support Wi-Fi.

Signed-off-by: Alex Zhao <zzc@rock-chips.com>
Change-Id: I987b5407d9f4a6fbae7766bb8a656f7c4a88e202
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index a6d4119983e1..9fcce13bd98f 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -775,6 +775,7 @@ CONFIG_USB_CONFIGFS_F_MIDI=y
 CONFIG_USB_CONFIGFS_F_UVC=y
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
+CONFIG_SDIO_KEEPALIVE=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_MMC_SDHCI_OF_ARASAN=y
-- 
2.35.3

